app/models/order.rb

```diff
@@ -89,6 +89,25 @@ def set_initial_state
     self.state ||= :draft
   end
 
+  def is_priority?
+    now = Time.now.in_time_zone
+    last_6pm = now.change(hour: 18, min: 0, sec: 0)
+    last_6pm -= 24.hours if now < last_6pm
+
+    case state
+    when "submitted" then
```

Comment:
then is not needed here. we can remove all `then`.

Reply:
Done